### DISCUSSION TITLE
Support for mantine v7

### DISCUSSION THREAD

**Discussion by Sajarin-M** [icflorescu/mantine-datatable#391] [Q&A]
Is it planned to support mantine v7 once released ?

**Comment by icflorescu** [comment]
That's what I'm hoping, but it most likely won't be a straightforward upgrade.
I'd very much like to do a deeper refactor and come up with a more improvements, but this is subject to whether I can get enough support from the community. As in sponsorship & code contributions :-)

**Comment by icflorescu** [comment]
### Mantine DataTable V7.1 was released on Nov 1st 2023 🎉

[![Mantine DataTable V7.1](https://github.com/icflorescu/mantine-datatable/assets/581999/72111cde-0404-467c-a333-aa68129b1a24)](https://icflorescu.github.io/mantine-datatable/)

If you're still using [Mantine V6](https://v6.mantine.dev/), stay on `mantine-datatable@6` and refer to the old documentation website that has been archived here: [icflorescu.github.io/mantine-datatable-v6](https://icflorescu.github.io/mantine-datatable-v6/)
